workers/workers.go

```diff
@@ -157,10 +170,15 @@ func (j *Job) Done(f func()) {
 	}
 }
 
+// Wait returns the value received by the j.result channel.
```

Comment:
Should indicate this only occurs once all tasks for a job are completed and after `Done` has been called.